lessons/Early-Math/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+= Evaluating Exponents
+
+@description{Students use Circles of Evaluation analyze the structure of expressions that include exponents.}
+
+@lesson-prereqs{absolute-value-and-opposites-2}
+
+@keywords{exponent, base}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Translate between words, mathematical expressions, and Circles of Evaluation.
+* Evaluate expressions involving exponents and one other operation.
+* Evaluate expressions involving exponents and negations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyze the structure of expressions with exponents.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Ensure that students are comfortable evaluating exponents during the Launch. Circles of Evaluation will help students parse expressions that involve exponents.
+
+|===
+
+== Expressions with Exponents and One Operation @duration{30 minutes}
+
+=== Overview
+
+Sudents use Circles, words, and mathematical notation to analyze the structure of expressions with exponents and one operation.
+
+=== Launch
+
+[.lesson-instruction]
+- Show students the following expression: @math{2^3}
+- One way of reading this expression is "two raised to the power of three." Do you know any other ways of translating this expression into words?
+** _Two to the third power; two to the third; two cubed_
+- What does it mean to _cube_ a number?
+** We multiply it by itself three times (literally, compouting the volume of a cube with a side-length of that number)
+- What does this expression evaluate to?
+** @math{2^3 = 2 \times 2 \times 2 = 8}
+
+Clarify and reinforce some key concepts about exponents via class discussion:
+
+- The number on the left is called the @vocab{base}. That number is multiplied by itself when we apply the exponent.
+
+- The smaller, raised number after the base is called the @vocab{exponent}; it indicates how many times to multiply the base.
+
+- "Cubing" is the same as "raising to the third power", and "squaring" is the same as raising to the second power.
+
+- There is no special terminology for any other exponents.
+
+Explain to students that we can represent expressions with exponents as Circles of Evaluation. The function we use to do so is called `num-expt`; when we use `num-expt`, the first value is the _base_, and the second value is the exponent.
+
+[.lesson-point]
+Exponents are valuable because they act as a shorthand.
+
+The Circle of Evaluation for @math{2 \times 2 \times 2 \times 2 \times 2} looks like this:
+
+[.centered-image]
+@show{(coe '(* (* (* (* 2 2) 2) 2) 2))}
+
+Here is the Circle of Evaluation for @math{2^5}, an equivalent expression:
```

Comment:
This section deserves a rewrite. 
I'd suggest consistently connecting expanded form (and its corresponding coe) with exponential form (and its corresponding coe). 

I also think the leap from 2^5 to num-expt(2 5) deserves some more care... and am curious whether the named function is worth introducing in a world without hands on keys...